datafusion-cli/Cargo.toml

```diff
@@ -26,7 +26,7 @@ license = "Apache-2.0"
 homepage = "https://datafusion.apache.org"
 repository = "https://github.com/apache/datafusion"
 # Specify MSRV here as `cargo msrv` doesn't support workspace version
-rust-version = "1.76"
+rust-version = "1.78"
```

Comment:
Due to ci error: 
error: package `aws-sdk-ssooidc v1.42.0` cannot be built because it requires rustc 1.78.0 or newer, while the currently active rustc version is 1.76.0\nEither upgrade to rustc 1.78.0 or newer, or use\ncargo update aws-sdk-ssooidc@1.42.0 --precise ver\nwhere `ver` is the latest version of `aws-sdk-ssooidc` supporting rustc 1.76.0\n" cmd="1.76.0-x86_64-unknown-linux-gnu cargo check"

Reply:
I also hit this in  https://github.com/apache/datafusion/pull/12032 and found another workaround: https://github.com/apache/datafusion/pull/12032/files#r1742677561 

(I don't think we can update the MSRV until Nov according to our policy

Reply:
Cool, I merged main, msrv check passed without bumping rust version.